sys-firmware/libkrunfw/libkrunfw-4.0.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Linux kernel shared object for libkrun"
+HOMEPAGE="https://github.com/containers/libkrunfw"
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~arm64"
+IUSE="asahi"
+
+MY_KVER="6.4.7"
+
+BDEPEND="
+	dev-python/pyelftools
+	asahi? ( app-alternatives/gzip )
+"
+
+SRC_URI="
```

Comment:
Put SRC_URI with DESCRIPTION/HOMEPAGE (see skel.ebuild's variable ordering, modulo whitespace, for what to do)